pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[tool.poetry]
+name = "gokart"
+version = "0.0.0"  # using poetry-dynamic-versioning
+description="A wrapper of luigi. This make it easy to define tasks."
+authors = ["M3, inc."]
+license = "MIT"
+readme = "README.md"
+homepage = "https://github.com/m3dev/gokart"
+repository = "https://github.com/m3dev/gokart"
+documentation = ""
+
+[tool.poetry-dynamic-versioning]
+enable = true
+style = "pep440"
+pattern = "^(?P<base>\\d+\\.\\d+\\.\\d+)"
+
+[tool.poetry.dependencies]
+python = "^3.6"
+luigi = "*"
+boto3 = "*"
+slack_sdk = "*"
+pandas = "*"
```

Comment:
for unit testing, we need below
`pandas = "<1.2"`

Reply:
@hirosassa Thanks
ref: https://github.com/pandas-dev/pandas/issues/39247